### PR TITLE
Use lua term

### PR DESCRIPTION
**Pull request by bkmgit**
https://github.com/meetecho/janus-gateway/issues/3272

### CODE REVIEW

File: src/plugins/lua/echotest.lua [hunk 1]
```diff
@@ -7,14 +7,14 @@ json = require('json')
 -- We also import our own SDP helper utilities: you may have better ones
 sdp = require('janus-sdp')
 -- Let's also use our ugly stdout logger just for the fun of it: to add
--- some color to the text we use the ansicolors library
--- (https://github.com/kikito/ansicolors.lua)
-colors = require "ansicolors"
+-- some color to the text we use the lua-trm library
```
Comment (lminiero) [inline src/plugins/lua/echotest.lua:10]: Typo here (trm instead of term).

File: src/plugins/lua/echotest.lua [hunk 2]
```diff
@@ -7,14 +7,14 @@ json = require('json')
...
+-- (https://github.com/hoelzro/lua-term)
+colors = require "term.colors"
```
Comment (lminiero) [inline src/plugins/lua/echotest.lua:12]: I've just noticed we're using a different syntax for this require, could you align it to the others?

File: src/plugins/lua/videoroom.lua [hunk 3]
```diff
@@ -7,14 +7,14 @@ json = require('json')
 -- We also import our own SDP helper utilities: you may have better ones
 sdp = require('janus-sdp')
 -- Let's also use our ugly stdout logger just for the fun of it: to add
--- some color to the text we use the ansicolors library
--- (https://github.com/kikito/ansicolors.lua)
-colors = require "ansicolors"
+-- some color to the text we use the lua-term library
+-- (https://github.com/hoelzro/lua-term)
+colors = require "term.colors"
```
Comment (lminiero) [inline src/plugins/lua/videoroom.lua:12]: Same here.

**Review by lminiero — COMMENTED** [review]
Thanks! Added just a couple of tiny editorial notes.

File: src/plugins/lua/echotest.lua (hunk 1 above)
Comment (bkmgit) [reply src/plugins/lua/echotest.lua]: Fixed.

File: src/plugins/lua/videoroom.lua (hunk 3 above)
Comment (bkmgit) [reply src/plugins/lua/videoroom.lua]: Fixed.

**Comment by lminiero** [comment]
I ported the same patch to `0.x` too.

**Pull request merged**